components/plan-build/bin/hab-plan-build.sh

```diff
@@ -957,16 +960,16 @@ _resolve_scaffolding_dependencies() {
   for dep in "${pkg_scaffolding}"; do
     _install_dependency "$dep"
     # Add scaffolding package to the list of scaffolding build deps
-    scaff_build_deps+=($dep)
+    scaff_build_deps+=("$dep")
     if resolved="$(_resolve_dependency "$dep")"; then
       build_line "Resolved scaffolding dependency '$dep' to $resolved"
-      scaff_build_deps_resolved+=($resolved)
+      scaff_build_deps_resolved+=("$resolved")
       # Add each (fully qualified) direct run dependency of the scaffolding
       # package.
-      sdeps=($(_get_deps_for "$resolved"))
+      sdeps=(_get_deps_for "$resolved")
       for sdep in "${sdeps[@]}"; do
-        scaff_build_deps+=($sdep)
-        scaff_build_deps_resolved+=($HAB_PKG_PATH/$sdep)
+        scaff_build_deps+=("$sdep")
+        scaff_build_deps_resolved+=("$HAB_PKG_PATH"/"$sdep")
```

Comment:
This is fine, but just FYI you can also do it like this and save a couple chars:
```suggestion
        scaff_build_deps_resolved+=("$HAB_PKG_PATH/$sdep")
```